src/lib/plugins/signs.js

```diff
@@ -1,7 +1,7 @@
 module.exports.server = (serv, { version }) => {
   const mcData = require('minecraft-data')(version)
 
-  const oakSignType = mcData.blocksByName.standing_sign.id
+  const oakSignType = mcData.blocksByName.sign.id
```

Comment:
this change will break backward compatibility, a `if (serv.supportFeature('theFlattening'))` should probably be added

Reply:
Yeah, expect other changes that may break stuff. I didnt have the time to check the code and look for them